mod/settings.php

```diff
@@ -390,7 +390,7 @@ function settings_post(App $a)
 			$err = true;
         }
 
-		if (User::isPasswordExposed($newpass)) {
+		if (!$a->getConfigValue('system', 'disable_password_exposed', false) && User::isPasswordExposed($newpass)) {
```

Comment:
Why not ```Config::get```?

Reply:
I only see one exception not to use the ```Config``` class: When you technically can't execute everything that is inside the class. Means: Since the ```Config``` itself uses database queries, we can't use this class inside the ```dba``` class. So it is a technical decision.

Here it is different. It is no technical decision not to use the class.

Reply:
I think you can always use the config script to set those values into the DB and not touch the .htconfig file at all.

Reply:
Indeed, I switched back to the `config` wrapper.